lib/clerk/rack_middleware_v2.rb

```diff
@@ -170,8 +170,19 @@ def call(env)
         return unknown(interstitial: true)
       end
 
-      if production? && client_uat.nil?
-        return signed_out(env)
+      # Show interstitial when there is no client_uat or cookie token
+      if (client_uat == "0" || client_uat.nil?) && cookie_token.to_s.empty?
```

Comment:
❓ That's the signed-out `client_uat` value, right? Just noting in case we need to update the comment accordingly.